packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -513,7 +517,7 @@ export class Relayer {
     transactions: { transaction: TransactionType | any; message: string; mrkdwn: string; level: string }[]
   ) {
     // Remove any undefined transaction objects or objects that contain null transactions.
-    transactions = transactions.filter((transaction) => transaction && transaction.transaction);
+    transactions = transactions.filter((transaction) => transaction && transaction !== null && transaction.transaction);
```

Comment:
This is an alternative bug fix, but we might as well add it as well.

Reply:
Good idea. Should watch out for this confusion in the future. Perhaps having a specific explicit null type.